robot_0:
  ns: quadrotor0
  frame: base_link
  num_sensors: 2
  sensor_0:       0
  sensor_1:       1
  sensor_explore: 2
  coll_rad:       1.0
  color_r:        0.8
  color_g:        0.0
  color_b:        0.0
robot_1:
  ns: quadrotor1
  frame: base_link
  num_sensors: 2
  sensor_0:       0
  sensor_1:       1
  sensor_explore: 2
  coll_rad:       1.0
  color_r:        0.0
  color_g:        0.8
  color_b:        0.0
robot_2:
  ns: quadrotor2
  frame: base_link
  num_sensors: 2
  sensor_0:       0
  sensor_1:       1
  sensor_explore: 2
  coll_rad:       1.0
  color_r:        0.0
  color_g:        0.0
  color_b:        0.8
